service/controller/v15/resource/cloudformation/create.go

```diff
@@ -145,6 +145,9 @@ func (r *Resource) newCreateChange(ctx context.Context, obj, currentState, desir
 		}
 
 		createState.SetTags(r.getCloudFormationTags(customObject))
+
+		enableTerminationProtection := key.EnableTerminationProtection
+		createState.EnableTerminationProtection = &enableTerminationProtection
 	} else {
```

Comment:
I am not sure if it can be done nicer in golang. I wanted to be defined as constant but I need to assign it to a variable to could reference, do I miss something?

Reply:
I think you should use `aws.Bool` here. AWS SDK is a bit weird and usually wants to receive pointers. aws.String is used a lot for this but there is a helper for bools too.

https://github.com/aws/aws-sdk-go/blob/master/aws/convert_types.go#L65